src/workflow/workflow.js

```diff
@@ -863,6 +825,68 @@ const verifyWorkflow = async (wfConfig) => {
   }
 };
 
+const calcMinInitialFund = async () => {
+  const context = getContext();
+  const { api, signingPair } = context.network;
+
+  let collectionId = 1;
+  let itemId = 1;
+  // calculate minimum initial fund
+  const destinationAddress = '5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY';
+  const { existentialDeposit } = api.consts.balances;
+  const info = await api.tx.uniques
+    .transfer(collectionId, itemId || 0, destinationAddress)
+    .paymentInfo(signingPair.address);
+  const fee = info.partialFee.muln(13).divn(10);
```

Comment:
Is there a better way to avoid having those magic 13 and 10 numbers?

Reply:
yeah!

Reply:
got it :) could you put that info as a comment?